examples/event_handler_rest/src/split_route_module.py

```diff
@@ -25,11 +25,7 @@ def get_todos():
 @router.get("/todos/<todo_id>")
 @tracer.capture_method
 def get_todo_by_id(todo_id: str):  # value come as str
-    api_key: str = router.current_event.get_header_value(
-        name="X-Api-Key",
-        case_sensitive=True,
-        default_value="",
-    )  # noqa: E501
+    api_key = router.current_event.headers["X-Api-Key"]
```

Comment:
How about using `.get()`? So we prevent the customer from having problems with `KeyError`.